Include Kokkos & OpenQASM backend in binary distribution (#198)
PennyLaneAI/catalyst: doc/changelog.md

# Release 0.2.1

<h3>Bug fixes</h3>

* Add missing OpenQASM backend in binary distribution, which relies on the latest version of the
  AWS Braket plugin for PennyLane to resolve dependency issues between the plugin, Catalyst, and

>   PennyLane. The Lightning-Kokkos backend with Serial and OpenMP modes is also added to the binary

Reviewer: We're not building against the SERIAL kernels anymore.

```suggestion
  PennyLane. The Lightning-Kokkos backend with OpenMP modes is also added to the binary

```

Reviewer: ```suggestion
  PennyLane. The Lightning-Kokkos backend with OpenMP kernels is also added to the binary

```

>   - Passes are now classes. This allows developers/users looking to change
> 
>     flags to inherit from these passes and change the flags.

Author: ```suggestion
  - Passes are now classes. This allows developers/users looking to change
    flags to inherit from these passes and change the flags.

```
